### ISSUE TITLE
about the predictor

### ISSUE THREAD

**Issue by yangyizu** [anthony-wang/CrabNet#6]
I'm a beginner . May I ask that  can your code predict the yield_strength of the material?

**Comment by anthony-wang** [comment]
Hi @yangyizu, yep we can! Please take a look in the data/matbench folder, there is the steels_yield dataset. Is this the dataset you want to predict on?
Or do you have your own data that you want to train / predict with?

**Comment by yangyizu** [comment]
In fact, I have been training the  steels_yield.csv dataset you provided, and I am trying to obtain a regression model with the method of deep learning to predict the yield_strength property. However, due to the small amount of data(the data set has just over 300 rows), the effect of my model is poor, and the MSE value is too large. So is there any way to solve my problem or is there any source code I can refer to? Thank you very much indeed！

**Comment by anthony-wang** [comment]
Could you please provide more details with what you are trying do?

In our source code and in the paper, we show how to train/evaluate CrabNet on the material properties (including steels_yield), so you can refer to the code there for those steps: https://github.com/anthony-wang/CrabNet/blob/4c46703d5f6e2e55e7280a507db117a586ad9af3/matbench_crabnet.py#L127-L134. One of the benefits of CrabNet is that it can also train and do well on small datasets like this one here.

Are you training steels_yield with CrabNet (then you can refer to the above), or with another neural network that you created?

**Comment by yangyizu** [comment]
I was given a task assigned by my teacher to predict the yield_strength of metal. The data set my teacher gave me is exactly the same as the data set you provided, maybe because our data sets are all from the same website. But the teacher's request is to only use the steels_yield.csv file for deep learning training, because we only need to predict the yield_strength. I tried a lot of common models, but the results were not good because the amount of data was too small. I will try your CrabNet model later. Thank you so much for your teaching！

**Comment by anthony-wang** [comment]
Hey, that sounds like a cool exercise!

So if the **requirement** is to use deep learning then you can either try a very simple MLP (multilayer perceptron, or a series of fully-connected layers) or you can also try using CrabNet.

In fact, we already have a trained CrabNet model using the steels_yield dataset, see: https://zenodo.org/record/4633866. 

But if you can use other machine learning model, maybe you can even use a simpler model like linear regression, random forests (like our comparison benchmark in the paper), gradient boosting, etc. These models typically do pretty well when the amount of data is small.

Let me know if you need more help!

**Comment by yangyizu** [comment]
thanks a lot！非常感谢！

**Comment by anthony-wang** [comment]
沒問題 🙂
